### PR TITLE
set new minimum browsers versions to run html5 client

### PR DESCRIPTION
**Pull request by Tainan404**
This PR updates the minimum browser version requirements to run the html5 client. I based this update on the following browser features used by the client: `Network Information`, `Page Visibility`, `WebSocket`, `MediaDevices` and `WebRTC`.

Closes #8521.

### CODE REVIEW

File: bigbluebutton-html5/private/config/settings.yml [hunk 1]
```diff
@@ -350,23 +350,23 @@ private:
...
       version: Infinity
     - browser: firefox
-      version: 52
+      version: 63
     - browser: firefoxMobile
-      version: 52
+      version: 68
     - browser: edge
-      version: 17
+      version: 72
```
Comment (capilkey) [inline bigbluebutton-html5/private/config/settings.yml]: Is Edge really reporting as 72?

**Comment by daminebenq** [comment]
tested **dev23** on **google-chrome-stable_70.XX.XX**: 
**NOTE:** Just Share Screen didn't work, it was asking for the share screen plugin

v70

File: bigbluebutton-html5/private/config/settings.yml (hunk 1 above)
Comment (Tainan404) [reply bigbluebutton-html5/private/config/settings.yml]: Yes
[Edge versions](https://en.wikipedia.org/wiki/Microsoft_Edge#Chromium_(2019%E2%80%93present))

**Comment by zorun** [comment]
What is the rationale for raising the minimum supported version on all these browsers? As shown by @daminebenq  BBB seems to work fine with older browser versions.

**Pull request merged**